packages/hash/api/src/db/postgres/client.ts

```diff
@@ -136,9 +136,9 @@ export class PostgresClient implements DbClient {
     });
   }
 
-  async getSystemTypeLatestVersion(params: {
-    systemTypeName: SystemType;
-  }): Promise<EntityType> {
+  async getSystemTypeLatestVersion(
+    params: Parameters<DbClient["getSystemTypeLatestVersion"]>[0],
+  ): Promise<EntityType> {
```

Comment:
To note that this could be `ReturnType<DbClient["getSystemTypeLatestVersion"]>>` so that both params and return automatically update if the adapter interface does. 

Doesn't need to be done as part of this PR but I notice you've updated all the params so thought I'd mention as a potential future improvement.

Reply:
Ah I had not considered that. In that case I'll add the reusable return types as well to ensure the PG adapter methods don't return more data than specified by their type.

Reply:
Addressed in https://github.com/hashintel/hash/pull/403/commits/aba7bc563ecb75c7b9a608bbce2a4179b0d2abd2